lib/themes/dosomething/paraneue_dosomething/includes/preprocess/preprocess_node.inc

```diff
@@ -196,14 +196,13 @@ function paraneue_dosomething_preprocess_node_campaign(&$vars) {
     // Add JS to open the signup data form modal.
     $id = 'modal-signup-data-form';
     $closeButton = 'false';
-    $skipForm = '#dosomething-signup-user-skip-signup-data-form';
 
     // Add skip button if setting is enabled
     if (isset($vars['node']->skippable_signup_data_form)) {
-      $closeButton = 'skip';
+      $closeButton = 'X';
```

Comment:
We use the "times" character for this elsewhere because it looks a little more clean: `×`